sdk-api-src/content/synchapi/nf-synchapi-acquiresrwlockshared.md

```diff
@@ -65,6 +65,10 @@ Acquires a slim reader/writer (SRW) lock in shared mode.
 
 A pointer to the SRW lock.
 
+## -Remarks
+
+Successful acquistition of a slim reader/writer (SRW) lock in shared mode generally permits other thread to acuire the same lock in shared mode, but doesn't guarantee that such acqustition would always succeed. SRW locks may limit its simultaneous shared mode acquistitions for performance reasons.
```

Comment:
It's spelled acquisition, acquire, acquisition and acquisitions.

Same goes for the other file.